components/psst/browser/core/psst_component_installer.cc

```diff
@@ -148,7 +172,7 @@ bool PsstComponentInstallerPolicy::IsBraveComponent() const {
 
 void RegisterPsstComponent(component_updater::ComponentUpdateService* cus) {
   if (!base::FeatureList::IsEnabled(psst::features::kBravePsst) || !cus) {
-    // In test, |cus| could be nullptr.
+    PsstComponentInstallerPolicy::DeleteComponent();
```

Comment:
do we actually need to delete this? In most cases components just skip registration here